app/controllers/admin/domain_blocks_controller.rb

```diff
@@ -17,7 +17,7 @@ class DomainBlocksController < BaseController
     PERMITTED_UPDATE_PARAMS = PERMITTED_PARAMS.without(:domain).freeze
 
     def batch
-      authorize :domain_block, :create?
+      authorize :domain_block, :index?
```

Comment:
I don't think this one is correct. This endpoint is for batch actions, so modifying the underlying blocks. I think `create?` is probably more accurate? (Though individual actions are checked in the form, so it does not matter that much)